settings.gradle

```diff
@@ -36,6 +36,8 @@ library("management")
 library("worldgen")
 library("misc")
 
+library("fluid")
```

Comment:
libraries should be in alphabetical order, so this should be after `library("entity")`